tap_salesforce/sync.py

```diff
@@ -103,7 +103,7 @@ def sync_stream(sf, catalog_entry, state):
             sync_records(sf, catalog_entry, state, counter)
             singer.write_state(state)
         except RequestException as ex:
-            raise Exception("Error syncing {}: {} Response: {}".format(
+            raise Exception("Error syncing {}: {} Response: {}".format(# pylint: disable=raise-missing-from
```

Comment:
If you're going to make a change here, the better action would probably be to add the `from` to it. like `raise Exception(...) from ex`

I think this was written before we did pylint and before we understood Python better. The `from` syntax makes the stack trace more readable.